_docs/demo/bookinfo.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: page
+title: Bookinfo
+permalink: /docs/demo/bookinfo/
+category: Demo
+order: 3
+---
+
+In this demo, we will deploy a simple app that displays information about a
+book, similar to a single catalog entry of an online book store. Displayed
+on the page is a description of the book, book details (ISBN, number of
+pages, and so on), and a few book reviews.
+
+The bookinfo application is broken into four separate microservices:
+
+* *productpage*. The productpage microservice calls the *details* and *reviews* microservices to populate the page.
+* *details*. The details microservice contains book information.
+* *reviews*. The reviews microservice contains book reviews. It also calls the *ratings* microservice.
+* *ratings*. The ratings microservice contains book ranking information that accompanies a book review. 
+
+There are 3 versions of the reviews microservice:
+
+* Version v1 doesn't call the ratings service.
+* Version v2 calls the ratings service, and displays each rating as 1 to 5 black stars.
+* Version v3 calls the ratings service, and displays each rating as 1 to 5 red stars.
+
+The end-to-end architecture of the application is shown below.
+
+![Bookinfo app](/docs/figures/amalgam8-example-app-bookinfo.svg)
+
+This application is polyglot, i.e., the microservices are written in
+different languages. All microservices are packaged with the
+[Amalgam8 Sidecar](/doc/sidecar/) that provides automatic service
+registration. For microservices that need to make outbound API
+calls, i.e., `productpage` and `reviews`, the sidecar acts as a proxy
+routing requests to appropriate instances of upstream services, based on
+the rules set forth by the Control Plane.
+
+### Goals of the demo
+
+* We will use Amalgam8's content-based routing feature to selectively
+enable `reviews v2` for a specific QA user named `jason`.
+
+* Using Amalgam8's systematic resilience testing feature, we will inject
+faults into the communication between microservices and test the failure
+recovery capability of the application. The impact of the failure will be
+restricted to only the QA user `jason`.
+
+* After fixing the bugs exposed by systematic resilience testing, we will
+gradually rollout a new version `reviews v3` similar to a canary rollout.
+
+## Deploy the App
+
+The commands to deploy the bookinfo demo application for different
+environments are as follows:
+
+_Docker Compose_
+  
+```bash
+docker-compose -f examples/docker-bookinfo.yaml up -d
+```
+
+_Kubernetes_ on localhost or on Google Cloud
+
+```bash
+kubectl create -f examples/k8s-bookinfo.yaml
+```
+
+_IBM Bluemix_
+
+1. Create Bluemix routes (DNS names) for the registry, controller and the bookinfo app's gateway:  
+
+   ```bash
+   cf create-route <your bluemix space> mybluemix.net -n <your bookinfo route>
```

Comment:
With the effort to provide a unified CLI in Bluemix this  "cf create-route <your bluemix space> mybluemix.net -n <your bookinfo route>"   should be    "bluemix cf create-route <your bluemix space> mybluemix.net -n <your bookinfo route>"

I have verified putting "bluemix" on the front of the command works.